__tests__/app.test.js

```diff
@@ -19,3 +19,37 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("should respond with 200 and get all reviews with an added key of comment count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews[5]).toEqual({
```

Comment:
could you refactor this to use a forEach - then check each object matches the shape you want, using .toMatchObject - or something similar?
This is a brittle test at the moment, is the reviews were to change